front_end/src/server.js

```diff
@@ -0,0 +1,33 @@
+var express = require('express');
+var path = require('path');
+var httpProxy = require('http-proxy');
+
+var proxy = httpProxy.createProxyServer().listen(3000);
```

Comment:
yeah, it would be for local dev. Otherwise you have to be rebuilding the bundle manually all the time and refreshing the browser and all that. For deploying, you would just create the distribution/bundle and it would get served by apache/nginx or whatever you are using currently.